### DISCUSSION TITLE
Powerwall 2 Web UI - has changed

### DISCUSSION THREAD

**Discussion by JonBulman** [vloschiavo/powerwall2#105] [Q&A]
Hi

I now see that this has been superseded and they ask you to login to Tesla One. 

Any workaround?

Cheers
Jon

**Comment by vls29** [comment]
My Powerwall2 has updated sometime overnight and I'm also affected.

**Comment by BJReplay** [comment]
Hi people, 

After you click on continue:

![image](https://github.com/user-attachments/assets/4a33689b-f761-4f8d-9f7a-7cf957b15b1e)

if you read the page carefully, you can see there is an upgrade later option:

![image](https://github.com/user-attachments/assets/9e665fae-6e9a-4d67-92b1-2d314ecd2258)

**Reply by emanuelbalea** [reply]
Mine is upgraded as well. Clicking upgrade later only lets you see basic diagnostics now, no installer mode.

**Reply by vls29** [reply]
Many thanks @BJReplay.  That "upgrade later" button is incredibly subtle and I only noticed after I'd sent off a whinge email to tesla energy EMEA support,  about removing the APIs 😂 (oops!)

Took me a good hour of trial and error to reset my password last night to a new value as I was using the last 5 chars of my serial number.  In case it helps someone else, there's a post here with the instructions https://teslamotorsclub.com/tmc/threads/unable-to-log-into-gateway-after-24-36-2-update-work-around-found.336050/
